### ISSUE TITLE
Default value fallback for Settings

### ISSUE THREAD

**Issue by Slayvin** [yii2-bridge/core#47]
The settings and settings groups are a great feature, but there is a problem when you have several language, and one is empty.
Let's say I want to use Settings to allow users to enter a GoogleMap api key and they have their sites in 3 languages.

**Current result :**
For now you have to fill the same value for the 3 languages in order to make it work. But for such setting, it makes no sense to have it translated, since the key is the same for all languages.

**Expected result**
If a settings value is empty/not set for a language,  it should fall back to the value of the default language.

**Comment by naffiq** [comment]
@Slayvin good point, that sounds like a small but valuable fix. Will you have time to fix this behavior or shall we assign it to somebody else?
